### DISCUSSION TITLE
Which elements in the periodic table are liquid at room temperature?

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#843] [Galaxy Trivia] [answered]
🌡️ 25°C / 77°F

**Comment by peterbaccaro** [comment, marked as answer]
At room temperature (around 25 degrees Celsius or 77 degrees Fahrenheit), only two elements on the periodic table are commonly liquid:

- **Mercury** (Hg): Mercury is a silvery metal that remains in a liquid state at room temperature. It is the only metal that exists as a liquid under standard conditions.

- **Bromine** (Br): Bromine is a non-metallic element that is a reddish-brown liquid at room temperature. It is highly volatile and evaporates easily, producing a distinctive odour.

These two elements, mercury and bromine, are the only elements that are liquid under typical room temperature conditions.
